src/ui/src/utils/env.tsx

```diff
@@ -28,8 +28,7 @@ const BUILD_TIMESTAMP = process.env.BUILD_TIMESTAMP;
 /* eslint-enable prefer-destructuring */
 
 const timestampSec = Number.parseInt(BUILD_TIMESTAMP, 10);
```

Comment:
This is called `timestampSec` but it's being multiplied as if it's milliseconds below. What is it really?

Reply:
the TIMESTAMP passed in is in seconds.
the date constructor and the date-fns formatters expect millis. hence we multiply by 1000 below.